packages/common/src/util.ts

```diff
@@ -124,63 +125,66 @@ exports.omit = (objectIn, keys) => {
  *
  * @alias module:util
  *
+ * @deprecated
+ *
  * @example
  * const isEven = (x) => x % 2 === 0;
  * const isOdd = negate(isEven);
  *
  * isOdd(2); // => false
  * isOdd(3); // => true
  */
-exports.negate = (predicate) => (...args) => !predicate.apply(this, args);
+// @ts-ignore
+export const negate = (predicate) => {
+  deprecate('@cumulus/common/util.omit()', '1.23.2', 'lodash/omit');
```

Comment:
Copy-paste error on the `deprecate` args?

Reply:
Thanks.